.circleci/config.yml

```diff
@@ -65,10 +65,10 @@ jobs:
   setup:
     executor: default-docker
     steps:
+      - checkout
```

Comment:
don't do checkout before installing dependencies -> for tagged branch builds (for branches different than master) the circle-ci default git client is failing